src/core/client/stream/tabs/Comments/Stream/PostCommentForm/CreateCommentMutation.ts

```diff
@@ -143,6 +149,12 @@ function tagUnansweredQuestions(
       featuredTag.setValue(GQLTAG.UNANSWERED, "code");
       node.setLinkedRecords(tags.concat(featuredTag), "tags");
     }
+  }
+
+  // Only manually increment Unanswered count when we're not actively on
+  // the Unanswered questions tab.
+  const local = lookup(environment, LOCAL_ID);
+  if (local.commentsTab !== "UNANSWERED_COMMENTS") {
```

Comment:
We don't the whole count logic here. Wyatt already manages counts in `incrementTagCommentCounts`, it just needs the right input.